developer-docs-site/docs/concepts/coin-and-token/Working with PropertyMap Off-Chain.md

```diff
@@ -0,0 +1,22 @@
+# Working with PropertyMap Off-Chain
+
+Token module uses property map [module](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-token/sources/property_map.move) to store on-chain properties of token. PropertyMap maps a string key to a property value on-chain, which stores the value in BCS serialized format and its type. Currently, only primitive types (bool, u8, u64, u128, address and String) are supported in property map. 
```

Comment:
Point to the token module and maybe call it The Aptos Token standard?

Reply:
should we update this to take advantage of recent new primitives in Move? u16, u32, u256?

Reply:
updated. Yes, we should. Let me create an issue in linear to track this issue and tagged you in the issue